[docs] add note around CRL rotation not occuring on revoke if auto_rebuild is enabled (#26893)
hashicorp/vault: website/content/api-docs/secret/pki.mdx

This endpoint revokes a certificate using its serial number. This is an
alternative option to the standard method of revoking using Vault lease IDs. A

> successful revocation will rotate the CRL (unless `auto_rebuild` is set to true
> per [the CRL configuration](#set-revocation-configuration)).

Reviewer: ```suggestion
successful revocation rotates the CRL unless `auto_rebuild` is set to `true`
 in [the CRL configuration](#set-revocation-configuration).
```
Style correction: write in active voice

This is an alternative option to the standard method of revoking using Vault
lease IDs or revocation via serial number. A successful revocation will

> rotate the CRL (unless `auto_rebuild` is set to true
> per [the CRL configuration](#set-revocation-configuration)).

Reviewer: ```suggestion
rotate the CRL unless `auto_rebuild` is set to true
in [the CRL configuration](#set-revocation-configuration).
```
